doc/collision_prediction_report.Rmd

```diff
@@ -52,23 +52,23 @@ knitr::include_graphics("../results/Distribution_of_fatality.png", dpi=100)
 Promising features in which we can see a clear difference in the shape of the distributions between collisions that are fatal and non-fatal include V_YEAR, C_MNTH, C_RCFG, and P_AGE. However, we decided to use all features in our model and to rely on recursive feature elimination with cross-validation to eliminate features that are not important for prediction.
 
 ```{r}
-CV_results |> 
+CV_results %>% 
```

Comment:
Why did you switch these?

Reply:
If it was because they no longer worked I think its because youre using an old version of R in the docker file